vocode/streaming/utils/worker.py

```diff
@@ -1,14 +1,14 @@
 from __future__ import annotations
 
 import asyncio
-import threading
-import janus
-from typing import Any, Optional
-from typing import TypeVar, Generic
 import logging
+import threading
+from typing import Any, Generic, Optional, TypeVar
 
+import janus
```

Comment:
doesn't look like we're using this anywhere?

Reply:
https://github.com/greymatter-labs/vocode-python-main/pull/400/files/6b4691e242cd98e0530eeca6904b4103eef011c2#r1896920564

Reply:
Also we are using this. 
https://github.com/greymatter-labs/vocode-python-main/blob/6b4691e242cd98e0530eeca6904b4103eef011c2/vocode/streaming/utils/worker.py#L54